migrations/versions/14e31b3a860b_.py

```diff
@@ -0,0 +1,38 @@
+"""empty message
+
+Revision ID: 14e31b3a860b
+Revises: 09c86e38f5e7
+Create Date: 2019-06-15 23:52:13.462368
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '14e31b3a860b'
+down_revision = '09c86e38f5e7'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('events', 'is_ngo')
+    op.alter_column('events_version', 'show_remaining_tickets',
```

Comment:
This seems to be occurring due to some issue with migration commands. Please remove this as we are not making any changes to the remaining tickets column in `events_version`

Reply:
@mrsaicharan1 is correct. This automatic script just saw that your events have a `null` `show remaining tickets` column.
That's why this line is being generated. so removing it should be fine.